### ISSUE TITLE
"The panorama can't be loaded" when switching too fast in VirtualTourPlugin

### ISSUE THREAD

**Issue by Skayo** [mistic100/Photo-Sphere-Viewer#608]
Hey there!
I'm using the virtual tour plugin with the "markers" mode, and I've noticed that the plugin throws an error if you switch the current node too fast (without waiting for animation to finish, etc.).
So to reproduce you basically just have to do:
```js
virtualTour.setCurrentNode('<first node>');
virtualTour.setCurrentNode('<second node>');
virtualTour.setCurrentNode('<third node>');
```
The problem is that two markers with the same ID can't exist at the same time, which happens when the plugin doesn't have time to finish removing the old markers.

Here's a demo: [JSFiddle](https://jsfiddle.net/n8tsu041/2/)

#### Versions:
- Photo Sphere Viewer: `4.4.1`
- three.js: `0.135.0`
- Browser: `Chrome 96.0.4664.110 (Official Build) (arm64)`

**Comment by mistic100** [comment]
Just discovered another error in the cancellation system #611, it will need to be fixed first.

**Comment by Skayo** [comment]
Thanks for the fix! Do you know when you'll release 4.4.2 because I really need that bug fix 😅
